server/graphql/v2/query/collection/AccountsCollectionQuery.ts

```diff
@@ -29,6 +29,14 @@ const AccountsCollectionQuery = {
       type: GraphQLBoolean,
       description: 'Only return Fiscal Hosts accounts if true',
     },
+    includeDeleted: {
+      type: GraphQLBoolean,
+      description: 'Included collectives which are soft deleted',
+    },
+    includeArchived: {
+      type: GraphQLBoolean,
+      description: 'Included collectives which are soft archived',
+    },
```

Comment:
:no_entry_sign: These params (at least `includeDeleted`) should be root only. We don't want random users to look at deleted entries in our DB.